src/main/java/school/lemon/changerequest/java/introduction/hw1/Task10.java

```diff
@@ -4,6 +4,20 @@
 public class Task10 {
 
     public static boolean containDigitTwo(int n) {
-        return false;
+        int i;
+        {
+            do {
+
+                i = n % 10;
+                if (i==2)  return true;
+                n=n/10;
+
+            }
+            while (n < 1);
+
+
+        }
+
+       return false;
```

Comment:
like this? 
https://github.com/dianasinenchenko/introduction.hw1/blob/94b65cdb91774687814ba410a8cc054f656d2dc4/src/main/java/school/lemon/changerequest/java/introduction/hw1/Task10.java

or maybe smth wrong